src/main/java/frc/robot/RobotContainer.java

```diff
@@ -490,85 +580,120 @@ private void configureButtonBindings() {
+    if (Robot.isSimulation() && intake != null) {
       return intake
           .humanPlayerIntake()
           .until(intake.pivotAtSetpoint(kPivotCoralStationAngle))
           .andThen(() -> intake.addGamePieceToIntakeSim())
           .asProxy();
-    } else {
+    } else if (intake != null) {
       return intake.humanPlayerIntake().until(intake.intakeHasCoralTrigger()).asProxy();
+    } else {
+      return Commands.none();
     }
   }
 
   public Command intakeFloorAutoCommand() {
-    if (Robot.isSimulation()) {
+    if (Robot.isSimulation() && intake != null && mapleSimArenaSubsystem != null) {
       return intake
           .floorIntake()
           .onlyWhile(() -> !mapleSimArenaSubsystem.getRobotHasCoral())
           .asProxy();
-    } else {
+    } else if (intake != null) {
       return intake.floorIntake().until(intake.intakeHasCoralTrigger()).asProxy();
+    } else {
+      return Commands.none();
     }
   }
 
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
   public Command scorerAutoCommand() {
-    if (Robot.isSimulation()) {
+    if (Robot.isSimulation() && intake != null && mapleSimArenaSubsystem != null) {
       return Commands.runOnce(() -> intake.removePieceFromIntakeSim())
           .andThen(mapleSimArenaSubsystem.scoreCoral())
           .until(() -> !mapleSimArenaSubsystem.getRobotHasCoral())
           .asProxy();
-    } else {
+    } else if (intake != null && coralScorer != null) {
       return coralScorer.scoreCommand().until(coralScorer.hasCoral().negate()).asProxy();
+    } else {
+      return Commands.none();
     }
   }
 
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
   public Command algeaRemoverAutoCommand() {
-    return algeaRemover
-        .removeAlgea()
-        .until(algeaRemover.algeaArmAtSetpoint())
-        .andThen(algeaRemover.goUp())
-        .asProxy();
+    if (algeaRemover != null) {
+      return algeaRemover
+          .removeAlgea()
+          .until(algeaRemover.algeaArmAtSetpoint())
+          .andThen(algeaRemover.goUp())
+          .asProxy();
+    } else {
+      return Commands.none();
+    }
   }
 
   public void givePreLoad() {
-    intake.addGamePieceToIntakeSim();
+    if (intake != null) {
+      intake.addGamePieceToIntakeSim();
+    }
   }
 
   public void startAuto() {
     if (Constants.currentMode != Constants.Mode.SIM) return;
 
-    driveSimulation.setSimulationWorldPose(drive.getPose());
+    if (drive != null && driveSimulation != null)
+      driveSimulation.setSimulationWorldPose(drive.getPose());
   }
 
   public void seedEncoders() {
-    intake.seedEncoder();
-    algeaRemover.seedEncoder();
-    climber.seedEncoder();
+    if (intake != null && algeaRemover != null && climber != null) {
+      intake.seedEncoder();
+      algeaRemover.seedEncoder();
+      climber.seedEncoder();
+    }
```

Comment:
Need these split out so they aren't all or nothing. 
e.g. If climber is null but intake isn't, the intake.seedEncoder() shouldn't still get skipped